app/packages/core/src/components/utils/editor/Editor.tsx

```diff
@@ -40,6 +40,9 @@ export const Editor: FunctionComponent<IEditorProps> = ({ language, readOnly = f
         fontFamily: 'monospace',
         readOnly: readOnly,
         scrollBeyondLastLine: false,
+        scrollbar: {
```

Comment:
The editor would otherwise "consume the scrollwheel". The user would have to move the mouse outside of the editor window to be able to resume scrolling the page. The user should never have to scroll inside the editor, because we set the height to match the line count. 
I think the change should be harmless for the other places where we use this editor. If not, I can make this an optional prop.